app/services/slot_availability.rb

```diff
@@ -29,7 +29,7 @@ def all_slots
 
 private
 
-  attr_reader :prison, :noms_id, :date_of_birth, :start_date, :end_date
```

Comment:
It's not changed, this change just prevent(ish) accessing the instance variable directly and encourages using accessor methods (initialize method).
I don't mind reverting it, in my opinion it's safer to mutate instance variables to through accessor methods.